vowpalwabbit/prob_dist_cont.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#include "vw.h"
+#include "v_array.h"
+#include "prob_dist_cont.h"
+#include "io_buf.h"
+
+using namespace std;
+// TODO: below check
+namespace VW { namespace actions_pdf
+{
+  void delete_prob_dist(void* v)
+  {
+    v_array<pdf_segment>* cs = (v_array<pdf_segment>*)v;
```

Comment:
Is this used as the prediction deletion function?

If so, please cast this to the union and then access the relevant field. This is to alleviate lots of issues when converting this union to something else :)

Reply:
Function deleted.